### PR TITLE
Change project structure to make it easy to use a maven repo like Jitpack.io

### PR DESCRIPTION
**Pull request by Hazer**
Hi,

I really liked your approach, and I am planning on playing with it, maybe contributing, so I would like to refer to it as an external dependency while not editing it directly.

I plan on working in some kind of orm for Android, one that don't try to eliminate SQL from the process and its more or less like a set of helpers above a great and safe DSL, and I really liked the DSL you are doing now.

Hope you don't mind =P

### CODE REVIEW

**Comment by x2bool** [comment]
Hi,

I am glad you like it. Yes, the project needs a Maven repo. My question is: Why use Jitpack and not jCenter for example? Actually I have never heard of Jitpack until now.

Would love to have contributions or feedback. Also keep me updated on your ORM progress, I am interested in some project like this myself.

File: local.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Mon Jan 04 14:10:54 BRST 2016
+ndk.dir=/Users/Hazer/Library/Android/sdk/ndk-bundle
```
Comment (x2bool) [inline local.properties]: This should not be in gut repo

Comment (Hazer) [inline local.properties]: Oops

**Comment by Hazer** [comment]
Hello,

Sorry for the delay on it.

Well, only because its easy to put it working for my needs, Jitpack makes easy to pick builds, I can choose a specific commit, tag, version or snapshot, and it will build from Github. It's quiet common in Android, I am using some libs published only there.

jCenter looks good for release, but I did not found a way to use jCenter for snapshot and pick commits, what I love doing. I was searching for it, and saw that for snapshots we can use the JFrog's OSS Artifactory or other service, so there is the need for plugins and more integrations.

Sooo, in the end, I made it compatible with both =D

Now you can publish to jCenter, and the script still works with Jitpack, any problem releasing your SNAPSHOT through Jitpack instead of another repository service?

To release to jCenter, just configure a maven.properties using the sample-maven.properties and your settings, I did not put it in local.properties, because actually I don't like mixing configurations, I can change it if you like.

You can use one of the publishConfigurations provided to upload to jCenter, there are 2, one for Core and one for SQLite module. I did not made a single command to publish all, maybe later.

What do you think?

**Comment by x2bool** [comment]
Hi,

It looks good to me. I am going to take a few days to review this. Thanks.

**Comment by x2bool** [comment]
Good job! Thank you.

**Pull request merged**